### ISSUE TITLE
pdf of Binomial(n, 1.)

### ISSUE THREAD

**Issue by chrodan** [JuliaStats/Distributions.jl#384]
There is a bug in the full pdf of the binomial distribution with success probability 1:
```julia
using Distributions
d = Binomial(5, 1.)
pdf(d)
```
gives
```
6-element Array{Float64,1}:
   0.0
 NaN  
 NaN  
 NaN  
 NaN  
 NaN 
```

The issue is the recursive evaluator in binomial.jl
```RecursiveBinomProbEvaluator(d::Binomial) = RecursiveBinomProbEvaluator(d.n, d.p / (1.0 - d.p))```
The problem occurs only in the range-version of the pdf, single queries work.

**Comment by johnmyleswhite** [comment]
Would be great to have a pull request for this.

**Comment by lindahua** [comment]
Looks like it has been fixed by #386.
